Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -53,7 +53,21 @@ final class DemoMerchantAPI {
             throw error
         }
     }
-    
+
+    func completeOrder(intent: Intent, orderID: String) async throws -> Order {
```

Comment:
When we refactor card we should use this singular method - the only difference between authorize and capture is the intent so we don't need 2 separate methods for that